Automattic/wp-calypso
# Remove simplified backup status page component

#### Changes proposed in this Pull Request
This PR removes an unused Simplified backup status page component as it was under feature flag and not used or will be in the furure.

#### Testing instructions
 - Create a new JN site
 - Do some modifications to the site to have several rewind points (install some plugins or update pages)
 - Spin up the changes locally `yarn start` and `yarn start-jetpack-cloud-p`
 - Visit `http://calypso.localhost:3000/backup/**Your JN site**` and check it functions properly
 - Visit `http://jetpack.cloud.localhost:3001/backup/**Your JN site**` and check it functions properly
